components/hab/habitat/plan.sh

```diff
@@ -72,6 +73,8 @@ do_prepare() {
   # package proper--it won't find its way into the final binaries.
   export LD_LIBRARY_PATH=$(pkg_path_for gcc)/lib
   build_line "Setting LD_LIBRARY_PATH=$LD_LIBRARY_PATH"
+
+  export JEMALLOC_SYS_DISABLE_STATIC=""
```

Comment:
Does
```suggestion
  export JEMALLOC_SYS_DISABLE_STATIC
```
or
```suggestion
  export JEMALLOC_SYS_DISABLE_STATIC="true"
```
work? In light of our misadventures with them, I find variables explicitly exported as the empty string to be confusing with regards to whether they intend to enable or *disable* behavior.

Reply:
Fair enough.  I'll try it both ways and update it if either works.

Reply:
It looks like `export JEMALLOC_SYS_DISABLE_STATIC` works so I'll go with that.